src/sentry/models/release.py

```diff
@@ -217,10 +217,14 @@ def filter_by_stage(
         operator: str,
         value,
         project_ids: Sequence[int] = None,
+        environments: List[str] = None,
```

Comment:
Should we be filtering by these envs as well? cc @iProgramStuff

Reply:
I think it could possibly help with performance of the query, but I think in terms of correctness I don't think it has an effect because we call `add_environment_to_queryset` in the endpoint which adds the env filtering